PennMobile/Home/Controllers/HomeViewController.swift

```diff
@@ -164,7 +164,7 @@ extension HomeViewController {
 
     func fetchCellData(for itemTypes: [HomeCellItem.Type], _ completion: (() -> Void)? = nil) {
         let items = tableViewModel.getItems(for: itemTypes)
-        self.fetchCellData(for: items)
+        self.fetchCellData(for: items, completion)
```

Comment:
Good catch!

Reply:
Haha thanks, that's why the refresh wouldn't ever end.